### PR TITLE
UI Update

### PR DESCRIPTION
**Pull request by blackstar257**

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+caprivacy.dagrizbox.com
```
Comment (caprivacy) [inline CNAME]: what is this for?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+
+link: Tell them to support 2FA
```
Comment (caprivacy) [inline index.html]: Haha might want to change this. Thankfully they use MIT license so we can just link to them in a license file as a way to provide a link back

**Review by caprivacy — COMMENTED** [review]
Just a couple of comments, looks amazing thanks! :)

File: CNAME (hunk 1 above)
Comment (blackstar257) [reply CNAME]: Fixed

File: index.html (hunk 2 above)
Comment (blackstar257) [reply index.html]: Removed

**Comment by blackstar257** [comment]
Before merging need to fix bug that could exist due caprivacy.me being a redirect and not a CNAME.

**Comment by blackstar257** [comment]
@caprivacy when you do this merge can you change the redirect you have for caprivacy.me to a CNAME record. I have included in this PR the CNAME file needed to make this work.

This is what it would be.
@ CNAME caprivacy.github.io

**Comment by caprivacy** [comment]
Yes thank you. I'll most likely merge tonight :)

File: CNAME [hunk 3]
```diff
@@ -0,0 +1 @@
+caprivacy.me
```
Comment (caprivacy) [inline CNAME]: why is this necessary to add a CNAME to my own root this way? Google domains which is where this is registered doesn't support it.

**Review by caprivacy — COMMENTED** [review]
TAL @ CNAME

File: CNAME (hunk 3 above)
Comment (blackstar257) [reply CNAME]: Doing the CNAME allows it to look cleaner. Yeah your right Google Domains doesn't support CNAME at root level. Im so used to using Cloudflare for everything.  Cloudflare supports it. Looks like twofactorauth.org is using cloudflare as well to take care of this. If you don't want to do cloudflare you could do a redirect from caprivacy.me to www.caprivacy.me which you can set as a CNAME.

Error message from google domains:
```
CNAME records for the root domain are not supported
```

Seems like other people have this issue.
https://serverfault.com/questions/617248/does-google-domains-support-cname-like-functionality-at-the-zone-apex


If you just want to keep it showing up as https://caprivacy.github.io/caprivacy/ let me know and i can update the asset references in the code to match that base.

Comment (blackstar257) [reply CNAME]: https://github.com/2factorauth/twofactorauth/blob/master/CNAME

**Comment by blackstar257** [comment]
Got it working with relativity. No need to change CNAME at this time
